btr-web/btr-main-app/utils/validation.ts

```diff
@@ -0,0 +1,12 @@
+/**
+ * Check if the input value is a valid percentage number from "1" to "100"
+ * Empty string is considered valid
+ * @param value the input string
+ */
+export function validatePercentage (value: string): boolean {
```

Comment:
Like mentioned above I think we want to save this value as a number so update this validation based on that